packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -263,13 +262,172 @@ function joinMeeting(meeting) {
+    //     <legend>Authentication with OAuth </legend>
+    //           <input id="login" title="login" type="submit" value="login">
+    //           <input id="logout" title="logout" type="submit" value="logout" disabled="disabled">
+    //           <p id="login-status">not logged in</p>
+    // `;
+
+    document.getElementById('authForm').appendChild(div);
+
+    // Now, let's set up oauth login
+    document.getElementById('credentials').addEventListener('submit', (event) => {
+    // let's make sure we don't reload the page when we submit the form
+      event.preventDefault();
+
+      // Initiate login to Webex account
+      webex.authorization.initiateLogin();
+    });
+
+    // Now, let's set up logout
+    document.getElementById('logout').addEventListener('click', (event) => {
+      // let's make sure we don't reload the page when we submit the form
+      event.preventDefault();
+
+      if (webex.canAuthorize) {
+        // if already authenticated, logout on click
+        webex.logout();
+      }
+    });
+  }
+
+  if (isPaste) {
+    const legend = document.createElement('legend');
+    const accessToken = document.createElement('input');
+    const webexInit = document.createElement('input');
+    const register = document.createElement('input');
+    const registerStatus = document.createElement('p');
+
+    legend.innerText = 'Authentication with Access Token';
+    registerStatus.innerText = 'not registered';
+    accessToken.setAttribute('id', 'access-token');
+    accessToken.setAttribute('name', 'accessToken');
+    accessToken.setAttribute('placeholder', 'Your access token');
+    accessToken.setAttribute('type', 'Text');
+    webexInit.setAttribute('id', 'webexInit');
+    webexInit.setAttribute('title', 'webex.init()');
+    webexInit.setAttribute('type', 'submit');
+    webexInit.setAttribute('value', 'webex.init()');
+    register.setAttribute('id', 'register');
+    register.setAttribute('title', 'register');
+    register.setAttribute('type', 'submit');
+    register.setAttribute('value', 'register');
+    registerStatus.setAttribute('id', 'register-status');
+    div.append(legend);
+    div.append(accessToken);
+    div.append(webexInit);
+    div.append(register);
+    div.append(registerStatus);
+
+
+    document.getElementById('authForm').appendChild(div);
+
+    // Let's wire our form fields up to localStorage so we don't have to
+    // retype things everytime we reload the page.
+    [
+      'access-token'
+    ].forEach((id) => {
+      const el = document.getElementById(id);
+
+      el.value = localStorage.getItem(id);
+      el.addEventListener('change', (event) => {
+        localStorage.setItem(id, event.target.value);
+      });
+    });
+
+    // add event handlers for webex.init() button
+    document.getElementById('webexInit').addEventListener('click', (event) => {
+      // let's make sure we don't reload the page when we want to initialize webex
+      event.preventDefault();
+
```

Comment:
Add a comment about why we are doing this:

```suggestion
// Re-initializing the sdk to authorize with the pasted access token
```